src/core/Memory.test.ts

```diff
@@ -1,12 +1,23 @@
-import {getValues, saveValues} from "./Memory";
+import {getValues, saveValues, storageKey} from "./Memory";
+
+beforeEach(() => {
+    localStorage.clear();
+})
 
 test("Test That Value Retrieved will Be Empty If not Set", () => {
     expect(getValues()).toEqual([]);
 })
 
-test("Test that Value Can Be Saved And Retrieved", () => {
+test('Test That Value Will Be Saved In LocalStorage', () => {
```

Comment:
Let's maybe start naming tests in some manner.
Maybe let's call it 'When saveValues is executed Then data should be saved in localstorage'?

Reply:
Shure :)